src/components/TeacherComponents/CopyrightNotice/CopyrightNotice.tsx

```diff
@@ -66,22 +66,24 @@ const PostAlbCopyright = (
   const { copyrightYear } = props;
   const year = new Date(copyrightYear).getFullYear();
   return (
```

Comment:
Component was throwing a hydration error on the lesson overview page due to link being rendered in a P tag